dash/include/dash/halo/HaloMatrixWrapper.h

```diff
@@ -277,16 +276,18 @@ class HaloMatrixWrapper {
    */
   void update_at(region_index_t index) {
     auto it_find = _region_data.find(index);
-    if(it_find != _region_data.end())
-      update_halo_intern(it_find->second, false);
+    if(it_find != _region_data.end()) {
+      update_halo_intern(it_find->second);
+      dart_wait_local(&it_find->second.handle);
+    }
   }
 
   /**
    * Initiates an asychronous halo region update for all halo elements.
    */
   void update_async() {
     for(auto& region : _region_data)
-      update_halo_intern(region.second, true);
+      update_halo_intern(region.second);
```

Comment:
Please add braces

Reply:
I hate braces for one statement loops/if's . But for you, i will change it ;)